File: C# - Typing/DTO.cs
using System;
using System.Collections.Generic;


public class DTO {
    public  bool      Bool  { get; set; }
    public  int       Num   { get; set; }
    public  string    Text  { get; set; }
    public  string    Opts  { get; set; }
    public  DateTime  Time  { get; set; }
}


var data = new List<DTO> {
    new DTO { Bool=true, Num=123, Text="text", Opts="AAA", Time=DateTime.Now },
    new DTO { Bool=true, Num=456, Text="text", Opts="BBB", Time=DateTime.Now },
    new DTO { Bool=true, Num=789, Text="text", Opts="CCC", Time=DateTime.Now }
};

File: Go - Typing/DTO.go
package main

import "time"


type DTO struct {
    Bool  bool      `json:"bool"`
    Num   int       `json:"num"`
    Text  string    `json:"text"`
    Opts  string    `json:"opts"`
    Time  time.Time `json:"time"`
}

func main() {
    data := []DTO {
        { Bool:true, Num:123, Text:"text", Opts:"AAA", Time:time.Now() },
        { Bool:true, Num:456, Text:"text", Opts:"BBB", Time:time.Now() },
        { Bool:true, Num:789, Text:"text", Opts:"CCC", Time:time.Now() },
    }
}